test/Dockerfile.test-distroless

```diff
@@ -0,0 +1,60 @@
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+### Dockerfile for building a distroless image for testing
+# Modified version of https://github.com/kubernetes/release/blob/master/images/build/distroless-iptables/distroless/Dockerfile
+
+ARG STAGE_DIR="/opt/stage"
+
+FROM debian:bullseye-slim as build
+ARG STAGE_DIR
+
+COPY test/build/stage-binaries-from-package.sh /
+COPY test/build/package-utils.sh /
+COPY test/build/stage-binary-and-deps.sh /
+
+RUN mkdir -p "${STAGE_DIR}" && \
+    /stage-binaries-from-package.sh "${STAGE_DIR}" conntrack \
+    ebtables    \
+    ipset       \
+    iptables    \
+    kmod		&& \
+	`# below binaries and dash are used by iptables-wrapper-installer.sh` \
+    /stage-binary-and-deps.sh "${STAGE_DIR}" /bin/dash \
```

Comment:
Ah that's a good idea, I added this.

Now the wrapper can be run like `iptables-wrapper install <path>` and it will create symlink pointing to itself in the given path.

I went with the path being an explicit argument to avoid having to auto detect where iptables is. Depending on how you build the image, `iptables` might not exist neither in `/sbin` or `/usr/sbin`. In that case, I would then need to start looking for `iptables-nft` or `iptables-legacy`, which adds a bit more complexity. Or create a fake `iptables` executable somwhere, which is equivalent to passing the path explicitly. Let me know what you think.